FastaLenFilter/FastaLenFilter.go

```diff
@@ -0,0 +1,63 @@
+// Copyright ©2017 The bíogo Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Filter sequences that are above a length cut-off
+package main
+
+import (
+	"fmt"
+	"io"
+	"github.com/biogo/biogo/alphabet"
+	"github.com/biogo/biogo/io/seqio/fasta"
+	"github.com/biogo/biogo/seq/linear"
+	"os"
+)
+
+
+var (
+	inf     = flag.String("inf", "test.fna", "input filename")
+	outf    = flag.String("outf", "test_gt2500bp.fna", "output filename")
+	minLen = flag.Int("minLen", 2500, "minimum sequence length cut-off (bp)")
+	help    = flag.Bool("help", false, "help prints this message.")
+)
+
+
+func main() {
+	flag.Parse()
+	if *help {
+		flag.Usage()
+		os.Exit(0)
+	}
+	f, err := os.Open(*inf)
+	  if err != nil {
+		  fmt.Fprintf(os.Stderr, "open input FASTA file: %v.", err)
+		  os.Exit(1)
+	  }
+	defer f.Close()
+	dnaf := fasta.NewReader(f, linear.NewSeq("", nil, alphabet.DNA))
+	of, err := os.Create(*outf)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "open output FASTA file: %v.", err)
+		os.Exit(1)
+	}
+	w := fasta.NewWriter(of, 60)
+	defer of.Close()
+	for {
```

Comment:
This can be more idiomatically said using the `seqio.Scanner` type.

```
sc := seqio.NewScanner(in)
for sc.Read() {
	s := sc.Seq()
	if s.Len() > *min {
		_, err := w.Write(s)
		if err != nil {
			log.Fatalf("failed to write sequence %q: %v", s.ID(), err)
		}
	}
}
err := sc.Error()
if err != nil {
	log.Fatalf("failed during read: %v", err)
}
```

Note that `_, err := w.Write(s)` can be spelled `_, err := fmt.Fprintf(out, "%60a\n", s)`. When writing to stderr, this is easier, though potentially slower.